scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -197,19 +240,8 @@ interface IState {
     isSidebarCollapsed: boolean;
 }
 
-const getTranslateAction = (getItem: () => IArticle): IAuthoringAction => ({
-    label: gettext('Translate'),
-    onTrigger: () => {
-        showModal(({closeModal}) => (
-            <TranslateModal
-                closeModal={closeModal}
-                article={getItem()}
-            />
-        ));
-    },
-});
-
 export class AuthoringIntegrationWrapper extends React.PureComponent<IPropsWrapper, IState> {
+    highlightsPopupOpen: boolean;
```

Comment:
I was about to say that it isn't used outside the component and we should make it `private`, but found that it isn't used anywhere at all so we should remove it ;D